player/screenshot.c

```diff
@@ -62,6 +64,7 @@ void screenshot_init(struct MPContext *mpctx)
     *mpctx->screenshot_ctx = (screenshot_ctx) {
         .mpctx = mpctx,
         .frameno = 1,
+        .log = mp_log_new(mpctx, mpctx->log, "screenshot")
```

Comment:
Can this allocation fail?

Reply:
I can't find an example of someone null-checking the newly allocated logger, so It can fail, but if it does, there are more important things to worry about.

Reply:
We should still check for the failure if possible.

Reply:
`mp_log_new` always returns an allocated `mp_log` so I don't think we need to worry about this.